### DISCUSSION TITLE
Cannot render images in markdown file

### DISCUSSION THREAD

**Discussion by iamsantanubanerjee** [thuliteio/doks#1192] [Q&A] [answered]
I am trying to render the image "01-linear-regression.png" on my website, but unable to do so. The folder structure, my markdown (line 34) and the error are all captured in the below screenshot. Kindly, help me figure this out.

![markdown-image-error](https://github.com/gethyas/doks/assets/59969215/4276b21b-2b19-4f96-8c59-5616a6d6ef39)

**Comment by h-enk** [comment, marked as answer]
Please see https://github.com/gethyas/images/issues/21#issuecomment-1914112602

**Reply by iamsantanubanerjee** [reply]
I tried followiing the steps (tallied both your comment and the Hugo images website instructions). The issue is still not fixed with a new error popping up (as below):

`Error: error building site: "/home/santanu/projects/iamsantanubanerjee.github.io/content/machine-learning/regression/linear-regression.md:1:1": /home/santanu/projects/iamsantanubanerjee.github.io/node_modules/@hyas/images/layouts/_default/_markup/render-image.html:155:12": execute of template failed: template: _default/_markup/render-image.html:155:12: executing "_default/_markup/render-image.html" at <$r.MediaType.SubType>: can't evaluate field MediaType in type string`

Here's a [link to the branch](https://github.com/iamsantanubanerjee/iamsantanubanerjee.github.io/tree/dev), if it helps you understand what I did wrong.

**Reply by h-enk** [reply]
Update Images to the latest (you're still using a previous version):

```
npm install @hyas/images@latest
```

Use one of the [resource locations](https://images.gethyas.com/docs/basics/resources/) properly — for example global resource:

```md
![Scatter plot of the dataset](images/01-linear-regression.png)
```

Then it works:

![Snag_18d423](https://github.com/gethyas/doks/assets/3902872/1e1f798b-6c84-4fff-9a84-10e1a04abc31)

**Reply by nopedawn** [reply]
Hi, I'm also having this error now when I try to build production to do a commit on github, but my previous commits were fine before

https://github.com/nopedawn/nopedawn.github.io/actions/runs/11701764443/job/32588525598

*edit: nvm.. it works now
